pylib/Stages/Reporter/IUDatabase.py

```diff
@@ -142,7 +142,7 @@ def execute(self, log, keyvals, testDef):
         metadata['platform_name'] = self._extract_param(testDef.logger, 'MTTDefaults', 'platform')
         trl = self._extract_param(testDef.logger, 'MTTDefaults', 'trial')
         if trl:
-            metadata['trial'] = 1
+            metadata['trial'] = trl
```

Comment:
Should we try to normalize this into a `0` or `1` before to gets to the server?
`metadata['trial'] = 1 if trl else 0`
(check my Python to make sure that ternary expression will work)